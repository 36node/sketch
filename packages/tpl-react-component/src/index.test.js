import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Component from "./index";

it("should renders correctly", () => {
  const tree = renderer.create(<Component>somthing</Component>).toJSON();
  expect(tree).toMatchSnapshot();
});
