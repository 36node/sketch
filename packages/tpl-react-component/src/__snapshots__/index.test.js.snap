// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
.c0 {
  font-size: 20px;
  color: red;
}

<div
  className="c0"
>
  Hello world 
  jam
   
</div>
`;
